### ISSUE TITLE
`read_time` with Bard sets throws string conversion error

### ISSUE THREAD

**Issue by mynetx** [statamic/cms#4343]
The `read_time` modifier cannot traverse into sets within Bard fields.
Instead, an _Array to string conversion_ error is shown.

Statamic 3.2.9 Pro
Laravel 8.62.0
PHP 8.0.9
jonassiewertsen/statamic-documentation 1.5.0
webographen/statamic-dynamic-token 1.0.0
withcandour/aardvark-seo 2.0.16

### Discussed in https://github.com/statamic/cms/discussions/4341

<div type='discussions-op-text'>

<sup>Originally posted by **austriker27** September 30, 2021</sup>
Hello! 

Has anyone gotten the `read_time` modifier working with bard when you have sets? 

I found this post for awhile ago but its not working for me (post is from V2):
https://statamic.com/forum/2970-read-time-from-bard-fieldtype

When I try this in antlers: 

```twig
{{ blog_content | flatten | join | strip_tags | read_time:180 }} min read
```

I get this error: 
`Error: Array to string conversion`

Inside the CMS, at the bottom of the bard field it does show the read time when Im using sets. Not sure if this is helpful but I looked at the [source code](https://github.com/statamic/cms/blob/a3d9bbd5ac9bbd4ffad0d9126dd75f4756b6c570/resources/js/components/fieldtypes/bard/BardFieldtype.vue#L125) and its using this package: https://www.npmjs.com/package/read-time-estimate</div>

**Comment by lukeagsmith** [comment]
I also have this problem.

**Comment by jasonvarga** [comment]
Should be resolved by #6226
